pkg/scheduler/framework/plugins/volumebinding/binder.go

```diff
@@ -149,7 +149,7 @@ type InTreeToCSITranslator interface {
 type SchedulerVolumeBinder interface {
 	// GetPodVolumes returns a pod's PVCs separated into bound, unbound with delayed binding (including provisioning)
 	// and unbound with immediate binding (including prebound)
-	GetPodVolumes(pod *v1.Pod) (boundClaims, unboundClaimsDelayBinding, unboundClaimsImmediate []*v1.PersistentVolumeClaim, err error)
+	GetPodVolumes(pod *v1.Pod) (boundClaims, unboundClaimsDelayBinding, unboundClaimsImmediate []*v1.PersistentVolumeClaim, allPVs map[string][]*v1.PersistentVolume, err error)
```

Comment:
pls update the comment, also we are not returning all pvs in general, only the ones that belong to storage classes of unbound PVCs with delayed binding. 

Also, pls create a new type to group all the return values into one struct apart from the error